package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we need this file?

Reply:
This file and also the .eslint* files are created automatically by the testing suite as soon as you run composer install